docs/openstack/manila_adoption.md

```diff
@@ -0,0 +1,110 @@
+# Manila adoption
+
+WIP
+
+## Prerequisites
+
+WIP
+
+## Procedure - Neutron adoption
```

Comment:
A bit of copypasta here and below, other than that LGTM.

Reply:
Thanks; resolved